app/Http/Controllers/AuthController.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Password;
+use Illuminate\Mail\Message;
+use Illuminate\Support\Facades\Log;
+use Tymon\JWTAuth\Exceptions\JWTException;
+
+use JWTAuth, Hash;
+
+use App\User;
+
+class AuthController extends Controller
+{
+    /**
+     * Register a user
+     *
+     * @param Request $request
+     * @return \Illuminate\Http\JsonResponse
+     */
+    public function register(Request $request)
+    {
+        $request->validate([
+            'email' => 'required|email|unique:users,email|max:255',
+            'name' => 'required|max:255'
+        ]);
+
+        $user = User::create ([
+            'name' => $request->input('name'),
+            'email' => $request->input('email'),
+            'password' => Hash::make($request->input('password'))
```

Comment:
Is this salted?

Reply:
Good question. Looks like it is: https://mnshankar.wordpress.com/2014/03/29/laravel-hash-make-explained/